general/networking/caddy.md

```diff
@@ -75,5 +75,6 @@ Note the trailing slash - that is optional, but recommended.
 
 ## Community Links
 
-- [Windows Reverse Proxy Guide for Caddy v2](https://old.reddit.com/r/jellyfin/comments/gdwe0s/windows_and_caddy_v2_reverse_proxy_guide)
-- [Windows Reverse Proxy Guide for Caddy v1](https://www.reddit.com/r/jellyfin/comments/ek8ugr/windows_reverse_proxy_guide)
+- [Windows plus DuckDNS Setup](https://forum.jellyfin.org/t/securing-your-jellyfin-install-https-with-caddy-server/282)
+- [Windows Guide for Caddy v2](https://old.reddit.com/r/jellyfin/comments/gdwe0s/windows_and_caddy_v2_reverse_proxy_guide)
+- [Windows Guide for Caddy v1](https://www.reddit.com/r/jellyfin/comments/ek8ugr/windows_reverse_proxy_guide)
```

Comment:
I'd say the subdomain here should be consistent, preferably `www` on both to have the user preference apply.